tests/integration/records/multi_db/test_records_table2table.py

```diff
@@ -106,7 +106,7 @@ def source2target(self):
     set_stream_logging()
 
     for source in DB_TYPES:
-        for target in DB_TYPES:
+        for target in ['mysql']:  # DB_TYPES:TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker